### PR TITLE
Added awesome-videos to the learn category

### PR DESCRIPTION
**Pull request by lucasviola**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -246,6 +246,7 @@ If you want to create your own list, please read the [create list instructions](
 - [CLI Workshoppers/Adventures](https://github.com/therebelrobot/awesome-workshopper)
 - [Learn to Program](https://github.com/karlhorky/learn-to-program)
 - [Speaking](https://github.com/matteofigus/awesome-speaking)
+- [Tech Videos](https://github.com/lucasviola/awesome-videos)
```
Comment (sindresorhus) [inline readme.md]: Why is not your repo and its heading `Tech Videos`?

Comment (lucasviola) [inline readme.md]: So, should I change the heading to Awesome Videos?

Comment (sindresorhus) [inline readme.md]: No, it's clearly limited to tech videos. I'm asking you to make the repo name and readme heading to reflect so too.

Comment (lucasviola) [inline readme.md]: Ok. Will do.

**Comment by lucasviola** [comment]
Pushed another commit with the changes.

**Pull request closed without merging**